src/identity/Identity.spec.ts

```diff
@@ -11,11 +11,11 @@ describe('Identity', () => {
     expect(alice.seedAsHex).toEqual('0x2f2f416c696365')
 
     expect(alice.address).toEqual(
-      '5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TmTd'
+      '5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu'
```

Comment:
why did the address change?

Reply:
![confused](https://user-images.githubusercontent.com/9762897/69650707-4992e780-106f-11ea-8467-7c44e3a5267e.gif)

Reply:
The checksum calculation seems to be different now